api/openapi-spec/swagger.json

```diff
@@ -10989,6 +10989,10 @@
           "description": "GMSACredentialSpecName is the name of the GMSA credential spec to use.",
           "type": "string"
         },
+        "hostProcess": {
+          "description": "HostProcess determines if a container should be run as a 'Host Process' container. This field is alpha-level and will only be honored by components that enable the WindowsHostProcessContainers feature flag. Setting this field without the feature flag will result errors when validating the Pod.",
```

Comment:
```suggestion
          "description": "HostProcess determines if a container should be run as a 'Host Process' container. This field is alpha-level and will only be honored by components that enable the WindowsHostProcessContainers feature flag. Setting this field without the feature flag will result in errors when validating the Pod.",
```

Reply:
Is this a generated file?  I thought if I updated the other values it would it would fix this. I ran `make clean && make generated_files` based on https://github.com/kubernetes/community/blob/4026287dc3a2d16762353b62ca2fe4b80682960a/contributors/devel/sig-architecture/api_changes.md#edit-version-conversions

Reply:
I typically run `make update`, but the same idea. Should be auto-generated.

Reply:
make update did the trick